common/config/ringpop.go

```diff
@@ -56,6 +56,8 @@ const (
 	// BootstrapModeDNS represents a list of hosts passed in the configuration
 	// to be resolved, and the resulting addresses are used for bootstrap
 	BootstrapModeDNS
+	// BootstrapModeDNSSRV represents a list of DNS hosts passed in the configuration to resolve host and ports, resulting in a list of addresses
```

Comment:
Nit: add more details about a use case -- it allows using different ports as each SRV record can use a different port.